addons/packages/antrea/0.11.3/bundle/config/overlay/antrea_overlay.yaml

```diff
@@ -169,44 +169,15 @@ metadata:
     kapp.k14s.io/disable-default-label-scoping-rules: ""
 
 
-#@overlay/match by=overlay.subset({"kind":"DaemonSet","metadata":{"name": "antrea-agent"}})
+#@overlay/match by=overlay.subset({"kind": "DaemonSet", "metadata": {"name": "antrea-agent"}})
 ---
-kind: DaemonSet
-metadata:
-  #@overlay/match missing_ok=True
-  annotations:
-    kapp.k14s.io/disable-default-label-scoping-rules: ""
```

Comment:
ditto

Reply:
Its still there. I think its the diff algorithm shows that..

Reply:
It's not there:
https://github.com/tnqn/community-edition/blob/caa72aa5431d94d2a0db1baf10fa7e49d5310bfc/addons/packages/antrea/0.11.3/bundle/config/overlay/antrea_overlay.yaml#L172-L183

Reply:
do you mean this part? maybe i miss your point.  https://github.com/tnqn/community-edition/blob/caa72aa5431d94d2a0db1baf10fa7e49d5310bfc/addons/packages/antrea/0.11.3/bundle/config/overlay/antrea_overlay.yaml#L162-L169